patches/native-messaging-portal.patch

```diff
@@ -46,17 +55,17 @@ index d4e9f1fbd0d76..b042f95744731 100644
 +#  - 2: auto (true for snap and flatpak or GTK_USE_PORTAL=1, false otherwise)
 +- name: widget.use-xdg-desktop-portal.native-messaging
 +  type: int32_t
-+  value: 2
++  value: 0
```

Comment:
Why is the default value changed there?

Reply:
That is what we have in phab, i just took the patch there, rebased and extracted it

Reply:
The default is 0 in Phabricator because the portal side has not landed yet. The patch here should default to 2, otherwise regressions such as https://bugzilla.mozilla.org/show_bug.cgi?id=1930119 will occur.

The rationale for the default pref value is at https://phabricator.services.mozilla.com/D140803?id=941456#anchor-inline-868586

I am fixing the default pref values in #101 and #102.